### PR TITLE
LL-5492 - dont display app install success if the user already has accounts

### PR DESCRIPTION
**Pull request by machard**
https://ledgerhq.atlassian.net/browse/LL-5492

### CODE REVIEW

File: src/renderer/screens/manager/AppsList/AppsList.js [hunk 1]
```diff
@@ -154,7 +156,7 @@ const AppsList = ({
         dispatch={dispatch}
         isIncomplete={isIncomplete}
         addAccount={addAccount}
-        disabled={update.length >= 1}
+        disabled={update.length >= 1 || hasAccounts}
```
Comment (juan-cortes) [inline src/renderer/screens/manager/AppsList/AppsList.js:159]: This also has the side effect of not showing the install success banner after I install ETH if I already have BTC accounts for instance, I don't know if this is something we want from product's perspective. To me the story reads as _"don't show the success banner _if coming from a fw update reinstall_"_ which is different. 🤔 

To me it should still show (once per manager session) after a successful install of a supported app.
cc @cthepot-ledger

**Review by juan-cortes — CHANGES_REQUESTED** [review]

File: src/renderer/screens/manager/AppsList/AppsList.js (hunk 1 above)
Comment (LFBarreto) [reply src/renderer/screens/manager/AppsList/AppsList.js:159]: yep you need to create a `hasCurrencyAccounts` of use the `currenciesSelector` to compare the list you get with what you installed :thinking:

**Review by LFBarreto — CHANGES_REQUESTED** [review]

File: src/renderer/screens/manager/AppsList/AppsList.js [hunk 2]
```diff
@@ -83,6 +85,10 @@ const AppsList = ({
   const { push } = useHistory();
   const { search } = useLocation();
   const reduxDispatch = useDispatch();
+  const currenciesAccountsSetup = useSelector(currenciesSelector);
+  const disableInstallBanner = every(state.installed, app =>
+    some(currenciesAccountsSetup, currency => currency.id === state.appByName[app.name].currencyId),
```
Comment (LFBarreto) [inline src/renderer/screens/manager/AppsList/AppsList.js:90]: should we disable if you already added one account from the installed list? :thinking:

**Review by LFBarreto — CHANGES_REQUESTED** [review]

**Review by LFBarreto — APPROVED** [review]
:+1:

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
@machard see my comment in https://ledgerhq.atlassian.net/browse/LL-5492

**Review by LFBarreto — APPROVED** [review]

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
@machard check my comment on https://ledgerhq.atlassian.net/browse/LL-5492?focusedCommentId=115287
The banner is still present after reinstalling the nano apps at the end of the firmware update

**Review by LFBarreto — APPROVED** [review]

**Review by stual-ledger — APPROVED** [review]
🚀 lgtm

**Pull request merged**